config/routes.rb

```diff
@@ -61,6 +62,7 @@
       jsonapi_resources :specific_tube_creations, except: %i[update]
       jsonapi_resources :state_changes, except: %i[update]
       jsonapi_resources :studies
+      jsonapi_resources :submission_pools
```

Comment:
Does any `except: ` want to be added here too, or is `immutable` in the resource enough?

Reply:
Other resources are not using `except` when they're immutable.  You only need it if the resource is not immutable and you don't want to be able to update existing resources.  It's interesting that you mention `delete` though.  We haven't been protecting against deletion and we haven't been testing those requests either.  Rather than handle it here, I think that's a bigger issue that could be considered for a future PR.